endpoints/getting-started-grpc/client/main.go

```diff
@@ -35,19 +35,26 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"io/ioutil"
 	"log"
 
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	pb "google.golang.org/grpc/examples/helloworld/helloworld"
 	"google.golang.org/grpc/metadata"
+
+	googleoauth2 "golang.org/x/oauth2/google"
```

Comment:
please leave this as it was. just "google"

Reply:
Done